### ISSUE TITLE
Issue with functions included via composer

### ISSUE THREAD

**Issue by maxarvidsson** [vimeo/psalm#182]
This is the smallest example I could make (three files):

composer.json:
```
{
    "autoload": {
        "files": [
            "src/autoload.php"
        ]
    }
}
```
src/autoload.php:
```
<?php

/**
 * @return string
 */
function myfunc()
{
    return "Hello, World";
}
```
src/test.php:
```
<?php

myfunc();
```

I run (with psalm installed globally via composer):
```
composer install
psalm --init
psalm
```
And I get:
```
INFO: MissingReturnType - src/autoload.php:6 - Method myfunc does not have a return type, expecting string
```
But I expect there to not be any issues.

**Comment by muglug** [comment]
Thanks for this – will have a fix momentarily

**Comment by maxarvidsson** [comment]
This fix seems to have broken php 5.6, it is trying to use a method introduced in php 7:
```
PHP Fatal error:  Call to undefined method ReflectionFunction::getReturnType() in /home/max/.composer/vendor/vimeo/psalm/src/Psalm/Checker/FunctionChecker.php on line 143
```

**Comment by muglug** [comment]
Sorry, fixed in cfbf256593af3d0e41fda9cc9f8ce223878a23a7
